Dockerfile

```
@@ -109,3 +109,15 @@ RUN : \
 RUN : \
     && cargo install cargo-deny \
     && cargo install sqlx-cli
+
+# 
+# Protobuf tools
+# 
+RUN : \
+    && 
+    BIN="/usr/bin" && \
+    VERSION="1.18.0" && \
+    curl -sSL \
+    "https://github.com/bufbuild/buf/releases/download/v${VERSION}/buf-$(uname -s)-$(uname -m)" \
+    -o "${BIN}/buf" && \
+    chmod +x "${BIN}/buf"
```

keeping the empty new line:

```suggestion
    chmod +x "${BIN}/buf"

```